### PR TITLE
[libopenmpt] Fix incorrect lib in pkg-config file.

### PR DESCRIPTION
**Pull request by Sibras**
The pkg-config file generated from the custom CMake script uses -lopenmpt whereas this port creates output libs with the name libopenmpt (note lib prefix).

This PR is required to fix any downstream project that use pkg-config to load in openmpt (such as ffmpeg which is currently broken due to this issue)

### CODE REVIEW

File: ports/libopenmpt/CMakeLists.txt [hunk 1]
```diff
@@ -79,6 +79,9 @@ set(prefix "${CMAKE_INSTALL_PREFIX}")
 set(exec_prefix [[${prefix}]])
 set(includedir [[${prefix}/include]])
 set(libdir [[${prefix}/lib]])
+file(READ "libopenmpt/libopenmpt.pc.in" contents)
+string(REPLACE "-lopenmpt" "-llibopenmpt" contents "${contents}")
+file(WRITE "libopenmpt/libopenmpt.pc.in" "${contents}")
```
Comment (dg0yt) [inline ports/libopenmpt/CMakeLists.txt:84]: - I guess this is wrong for non-windows. (Basically, upstream pc file is right most of the times. vcpkg's vendor CMake build systems are technical dept.)
- Do not modify files in the source dir. Do the fixup in the binary dir or in the installed location.

Comment (Sibras) [reply ports/libopenmpt/CMakeLists.txt:84]: The main issue is that this port has its own custom CMakeLists.txt build file that sets the project name to 'libopenmpt' whereas the upstream openmpt build scripts use 'openmpt'. Thus the source 'libopenmpt.pc.in' (provided by upstream) uses the name 'openmpt'.

The correct thing to do would be to change the project name to 'openmpt' then everything would match upstream build scripts, but that would potentially break downstream vcpkg users who are explicitly looking for "libopenmpt.lib" etc.

Comment (Sibras) [reply ports/libopenmpt/CMakeLists.txt]: I Changed the fixup to the binary dir

**Comment by Sibras** [comment]
Closed as its not correct non-windows.

**Pull request closed without merging**